src/ui/toolbar/components/toolbar-input.vue

```diff
@@ -31,15 +30,26 @@ export default {
             uid: `mct-input-id-${inputUniqueId}`
         };
     },
+    mounted() {
+        if (this.options.type === 'number') {
+            this.$el.addEventListener('input', this.onInput);
```

Comment:
@psarram What do you think of using Vue events @input and @change instead of adding them onMounted and removing them onDestroy? Vue would take care of removing and adding those handlers.

Reply:
You could add conditional event listeners like this:
<input v-on="{ input: options.type === 'number' ? onInput : null }"/>

Reply:
@psarram nevermind - i tested the latest vue and this is very laggy because it reacts to onChange and reimplements the input handlers. Your approach is better!